GetDisplayUserInfo-McGuire/Program.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace GetDisplayUserInfo_McGuire
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Hey there! I just need some basic info!");
+
+            string firstName = GetStringUserInfo("What is your first name?");
+            string lastName = GetStringUserInfo("What is your last name?");
+            int favoriteNumber = GetIntUserInfo("What is your favorite number?");
+            int numberOfPets = GetIntUserInfo("How many pets do you have?");
+            List<string> nameOfPets = PetNames("What are the names of your pets?", numberOfPets);
+            
+
+            Console.WriteLine("Your name is " + firstName + " " + lastName + ".");
+            Console.WriteLine("Your favorite number is " + favoriteNumber + ".");
+            Console.WriteLine("You have " + numberOfPets + " pets.");
+            Console.WriteLine("Your pet names are: ");
+            foreach (var name in nameOfPets)
+            {
+                Console.WriteLine(name);
+            }
+
+
+        }
+
+        static string GetStringUserInfo(string question)
+        {
+            Console.WriteLine(question);
+            string answer = Console.ReadLine();
+            return answer;
+        }
+
+        static int GetIntUserInfo(string question)
+        {
+            Console.WriteLine(question);
+            int answer = Int32.Parse(Console.ReadLine());
```

Comment:
This could also be written as `int.Parse` instead of `Int32.Parse`. As an improvement, could you make it so that a non-integer response from the user doesn't cause an exception to be thrown? Probably `int.TryParse`